src/DynamoDbQueue.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace pixelandtonic\dynamodb;
+
+use Aws\DynamoDb\Exception\DynamoDbException;
+use Yii;
+use yii\base\InvalidConfigException;
+use yii\di\Instance;
+use yii\queue\cli\Queue;
+
+class DynamoDbQueue extends Queue
+{
+    public DynamoDbConnection|string|array $dynamoDb = 'dynamoDb';
```

Comment:
Same type comment/question as before. Would be better to have the complete class unless we can't for some strange reason.

Reply:
See explanation above